bundles/redhat8.10/README.md

```diff
@@ -0,0 +1,11 @@
+# Updating packages.txt.gotmpl
+
+The package list includes packages required by the NVIDIA GPU drivers, kernel- headers, and kernel-devel. These headers need to match the kernel version in the base image.
```

Comment:
```suggestion
The package list includes packages required by the NVIDIA GPU drivers, kernel-headers, and kernel-devel. These headers need to match the kernel version in the base image.
```

Reply:
`kernel-headers` and `kernel-devel` are the package names.

Reply:
Sorry--misread the diff! I see the extra space. I'll fix this in a follow-up :smile: